core/client/src/blockchain.rs

```diff
@@ -24,6 +24,19 @@ use sr_primitives::Justification;
 
 use crate::error::{Error, Result};
 
+/// Light header. Used to efficiently traverse the tree.
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct LightHeader<Block: BlockT> {
+	/// Hash of the header.
+	pub hash: Block::Hash,
+	/// Block number.
+	pub number: NumberFor<Block>,
+	/// Hash of parent header.
+	pub parent: Block::Hash,
+	/// Hash of an ancestor header. Used to jump through the tree.
+	pub ancestor: Block::Hash,
```

Comment:
for the user of the light-header API, is this `ancestor` field actually useful? Seems like it will change all the time. Best to hide such implementation details